jsf/subsystem/src/test/java/org/jboss/as/jsf/deployment/JSFModuleIdFactoryTestCase.java

```diff
@@ -52,25 +54,44 @@ public void validSlotTest() {
     @Test
     @Ignore("Depends on https://issues.redhat.com/browse/WFLY-17405")
     public void modIdsTest() {
-        Assert.assertEquals(API_MODULE, factory.getApiModId("main").getName());
-        Assert.assertEquals("main", factory.getApiModId("main").getSlot());
-        Assert.assertEquals(IMPL_MODULE, factory.getImplModId("main").getName());
-        Assert.assertEquals("main", factory.getImplModId("main").getSlot());
-        Assert.assertEquals(INJECTION_MODULE, factory.getInjectionModId("main").getName());
-        Assert.assertEquals("main", factory.getInjectionModId("main").getSlot());
-
-        Assert.assertEquals(API_MODULE, factory.getApiModId("myfaces").getName());
-        Assert.assertEquals("myfaces", factory.getApiModId("myfaces").getSlot());
-        Assert.assertEquals(IMPL_MODULE, factory.getImplModId("myfaces").getName());
-        Assert.assertEquals("myfaces", factory.getImplModId("myfaces").getSlot());
-        Assert.assertEquals(INJECTION_MODULE, factory.getInjectionModId("myfaces").getName());
-        Assert.assertEquals("myfaces", factory.getInjectionModId("myfaces").getSlot());
-
-        Assert.assertEquals(API_MODULE, factory.getApiModId("myfaces2").getName());
-        Assert.assertEquals("myfaces2", factory.getApiModId("myfaces2").getSlot());
-        Assert.assertEquals(IMPL_MODULE, factory.getImplModId("myfaces2").getName());
-        Assert.assertEquals("myfaces2", factory.getImplModId("myfaces2").getSlot());
-        Assert.assertEquals(INJECTION_MODULE, factory.getInjectionModId("myfaces2").getName());
-        Assert.assertEquals("myfaces2", factory.getInjectionModId("myfaces2").getSlot());
+        // Canonicalize module names using ModuleIdentifierUtil
+        String mainCanonical = canonicalModuleIdentifier("main", "main");
+        String myfacesCanonical = canonicalModuleIdentifier("myfaces", "myfaces");
+        String myfaces2Canonical = canonicalModuleIdentifier("myfaces2", "myfaces2");
```

Comment:
These 3 calls are not exercising the intended use of the API, which is combining a module name and a slot. In none of these 3 is the first param a module name.

The last two return 'myfaces:myfaces' and 'myfaces2:myfaces2', neither of which will work in the various uses below.

All of these can be removed.